asyn/drvAsynSerial/drvAsynSerialPortWin32.c

```diff
@@ -224,27 +224,27 @@ static void monitorComEvents(void* arg)
 			}
 		}
 		getTimestamp(datetime, sizeof(datetime));
-		if (evtMask & EV_ERR)
+		if ((evtMask & EV_ERR) && (tty->commEventMask & EV_ERR))
```

Comment:
We probably should have written at least a comment in the code explaining what's going on here too...

Reply:
not too worried myself about which of   `a & b & c` or  `(a & b ) && (c & b)` but yes a comment about the device and  why you need to re-check would be helpful

Reply:
Have added a couple of sentences explaining the reasoning behind this change.